Intersect.Server/Entities/Entity.cs

```diff
@@ -812,7 +819,7 @@ public virtual float GetMovementTime()
             var time = 1000f / (float) (1 + Math.Log(Stat[(int) Stats.Speed].Value()));
             if (Blocking)
             {
-                time += time * Options.BlockingSlow;
+                time += time * (float)Options.BlockingSlow;
```

Comment:
What effect does this change have?

Reply:
I think I saw that on the client side it was like this, so I left both the same, client and server, nothing really changes